### PR TITLE
Adds new policies/regos for AWS serverless services 

### PR DESCRIPTION
**Pull request by cesar-rodriguez**
This PR adds a few regos for AWS serverless services including: lambda, cloudwatch, and API Gateway

### CODE REVIEW

File: pkg/policies/opa/rego/aws/aws_lambda_function/AWS.LambdaFunction.EncryptionandKeyManagement.High.0471.json [hunk 1]
```diff
@@ -5,7 +5,7 @@
         "prefix": ""
     },
     "severity": "High",
-    "description": "Lambda does not have KMS key to protect environment variables.",
+    "description": "Lambda does not uses KMS CMK key to protect environment variables.",
```
Comment (williepaul) [inline pkg/policies/opa/rego/aws/aws_lambda_function/AWS.LambdaFunction.EncryptionandKeyManagement.High.0471.json:8]: use?

File: pkg/policies/opa/rego/aws/aws_lambda_function/AWS.LambdaFunction.Logging.Low.0470.json [hunk 2]
```diff
@@ -6,7 +6,7 @@
     },
     "severity": "LOW",
     "description": "Lambda tracing is not enabled.",
-    "reference_id": "AWS.LambdaFunction.Logging.Medium.0470",
+    "reference_id": "AWS.LambdaFunction.Logging.Low.0470",
```
Comment (williepaul) [inline pkg/policies/opa/rego/aws/aws_lambda_function/AWS.LambdaFunction.Logging.Low.0470.json:9]: I think it's best to leave the severity out of the reference_id completely... The newer policies should be doing this; the older ones still need to be scrubbed.

File: pkg/policies/opa/rego/aws/aws_lambda_function/AWS.LambdaFunction.Logging.Low.0472.json [hunk 3]
```diff
@@ -4,9 +4,9 @@
     "template_args": {
         "prefix": ""
     },
-    "severity": "Medium",
+    "severity": "Low",
     "description": "Lambda function doesn't not include a VPC configuration.",
-    "reference_id": "AWS.LambdaFunction.Logging.High.0472",
+    "reference_id": "AWS.LambdaFunction.Logging.Low.0472",
```
Comment (williepaul) [inline pkg/policies/opa/rego/aws/aws_lambda_function/AWS.LambdaFunction.Logging.Low.0472.json:9]: Same here w/severity in the reference_id

File: pkg/policies/opa/rego/aws/aws_cloudwatch/AWS.CloudWatch.EncryptionandKeyManagement.High.0632.json [hunk 4]
```diff
@@ -0,0 +1,10 @@
+{
+    "name": "logGroupNotEncryptedWithKms",
+    "file": "logGroupNotEncryptedWithKms.rego",
+    "template_args": null,
+    "severity": "HIGH",
+    "description": "AWS CloudWatch log group is not encrypted with a KMS CMK",
+    "reference_id": "AWS.CloudWatch.EncryptionandKeyManagement.High.0632",
```
Comment (williepaul) [inline pkg/policies/opa/rego/aws/aws_cloudwatch/AWS.CloudWatch.EncryptionandKeyManagement.High.0632.json:7]: Same comment about reference_id

File: pkg/policies/opa/rego/aws/aws_apigatewayv2_stage/AWS.ApiGatewayV2Stage.Logging.Low.0630.json [hunk 5]
```diff
@@ -0,0 +1,10 @@
+{
+    "name": "apiGatewayNoAccessLogs",
+    "file": "apiGatewayNoAccessLogs.rego",
+    "template_args": null,
+    "severity": "Low",
+    "description": "AWS API Gateway V2 Stage is missing access logs",
+    "reference_id": "AWS.ApiGatewayV2Stage.Logging.Low.0630",
```
Comment (williepaul) [inline pkg/policies/opa/rego/aws/aws_apigatewayv2_stage/AWS.ApiGatewayV2Stage.Logging.Low.0630.json:7]: reference id

File: pkg/policies/opa/rego/aws/aws_apigatewayv2_api/AWS.ApiGatewayV2Api.AccessControl.Medium.0630.json [hunk 6]
```diff
@@ -0,0 +1,10 @@
+{
+    "name": "apiGatewayMiconfiguredCors",
+    "file": "apiGatewayMiconfiguredCors.rego",
+    "template_args": null,
+    "severity": "Medium",
+    "description": "Insecure Cross-Origin Resource Sharing Configuration allowing all domains",
+    "reference_id": "AWS.ApiGatewayV2Api.AccessControl.High.0630",
```
Comment (williepaul) [inline pkg/policies/opa/rego/aws/aws_apigatewayv2_api/AWS.ApiGatewayV2Api.AccessControl.Medium.0630.json:7]: reference id

File: pkg/policies/opa/rego/aws/aws_apigatewayv2_api/apiGatewayMiconfiguredCors.rego [hunk 7]
```diff
@@ -0,0 +1,8 @@
+package accurics
+
+apiGatewayMiconfiguredCors[api.id] {
+  api := input.aws_apigatewayv2_api[_]
+  cors := api.config.cors_configuration[_]
+  origins := cors.allow_origins[_]
+  not origins == ["*"]
```
Comment (williepaul) [inline pkg/policies/opa/rego/aws/aws_apigatewayv2_api/apiGatewayMiconfiguredCors.rego:7]: if "origins" does not exist, does it default to all domains?

**Review by williepaul — CHANGES_REQUESTED** [review]

File: pkg/policies/opa/rego/aws/aws_apigatewayv2_api/apiGatewayMiconfiguredCors.rego (hunk 7 above)
Comment (cesar-rodriguez) [reply pkg/policies/opa/rego/aws/aws_apigatewayv2_api/apiGatewayMiconfiguredCors.rego:7]: No. It defaults to only trust requests coming from the domain your API Gateway is serving traffic from.

**Review by williepaul — APPROVED** [review]

**Pull request merged**